src/glass/src/app/pages/welcome-page/welcome-page.component.html

```diff
@@ -0,0 +1,81 @@
+<div class="glass-welcome-page"
+     fxLayout="column"
+     fxLayoutAlign="center center">
+  <div class="card">
+    <div class="card-header">
+      <div class="card-title"
+           *ngIf="!checked"
+           translate>
+        Welcome to Aquarium
+      </div>
+      <div class="card-title"
+           *ngIf="checked"
+           translate>
+        Requirements report
+      </div>
+    </div>
+    <div class="card-body">
+      <div class="card-text"
+           *ngIf="!checked"
+           translate>
+        In order to get this node started and ensure anything works as smoothly as possible, we need to check if it
+        fulfills at least a minimum set of requirements.
```

Comment:
```suggestion
    Before we begin, the following minimum set of requirements need to be met:

Reply:
If I have misunderstood, then this sentence needs to clarify: what "it fulfils" is

Reply:
The sentence is just an introduction in my opinion. By "it fulfils" I mean "check if the current node meets a minimum set of requirements".

There is no list of requirements on the first page at the moment (see screen below). Does it make sense to rephrase you sentence to e.g.: `Before we begin, check if the current node meets a minimum set of requirements.`?

![Screenshot_2021-10-05_14-24-25](https://user-images.githubusercontent.com/8761082/136153764-16556023-b925-41f7-8ce9-6ffb69761d23.png)